docker-compose.yml

```diff
@@ -12,13 +12,9 @@ services:
       - redis
 
     environment:
-      - BOT_TOKEN
-      - BOT_DEBUG
-      - BOT_GUILD
-      - BOT_ADMIN_ROLE_ID
-      - CHANNEL_DEVLOG
-      - CHANNEL_COMMUNITY_BOT_COMMANDS
       - REDIS_HOST=redis
+    env_file:
+      - .env
```

Comment:
`docker-compose up` already loads a `.env` automatically if it exists in the same directory as the compose file, but I suppose being explicit isn't bad.

Reply:
I haven't been able to produce this behaviour. Having the `.env` file in the root and running `docker-compose up` doens't seem to load the variables automatically